src/moss_price/main.py

```diff
@@ -0,0 +1,99 @@
+import os
+import json
+from web3 import Web3
+import discord
+from discord.ext import commands, tasks
+
+BOT_TOKEN = os.environ["DISCORD_BOT_TOKEN"]
+MCO2_ADDRESS = '0xfc98e825a2264d890f9a1e68ed50e1526abccacd'
+
+# Initialized Discord client
+intents = discord.Intents.all()
```

Comment:
sorry I missed this the first time around: could we change this to `default` instead of `all`? I'd actually like to change all of the price bots to use `default` since it's not necessary to elect all intents (as demonstrated on the rebase countdown bot: https://github.com/KlimaDAO/discord-bots/blob/main/src/next_rebase/main.py#L17)